app/assets/stylesheets/mobile/header.css.scss

```diff
@@ -169,10 +163,19 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
-       -o-transform: translateX(-80%);
-      -ms-transform: translateX(-80%);
-     -moz-transform: translateX(-80%);
-  -webkit-transform: translateX(-80%);
+.draw {
+    #drawer, #main_nav {
+            transform: translateX(-80%);
+    -webkit-transform: translateX(-80%);
+    }
+}
+
+#drawer, #main_nav {
+    z-index: 10;
+
+            transition: transform 0.3s ease;
+    -webkit-transition: -webkit-transform 0.3s ease;
+             transform: translateX(0);
+     -webkit-transform: translateX(0);
```

Comment:
Why do you need to "translate back"? When the .draw class is removed, the translate(-80%) is removed too, so the drawer should go back to his place.